pyhuman/app/workflows/search_web.py

```diff
@@ -2,9 +2,11 @@
 from time import sleep
```

Comment:
Should we call this workflow 'google_search' or something different.?

I infer the same functionality from the workflows "search_web.py" and "browse_web.py"